internals/plan/plan.go

```diff
@@ -26,6 +26,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/canonical/pebble/internals/logger"
```

Comment:
Nit: import ordering - this should go in the `canonical/pebble` section below.

Reply:
Is it okay to add logging to `ParseLayer`? Is there any reason the plan package didn't include any logging previously?